src/View/Helper/FormHelper.php

```diff
@@ -1044,6 +1047,19 @@ public function control(string $fieldName, array $options = []): string
         $options = $this->_parseOptions($fieldName, $options);
         $options += ['id' => $this->_domId($fieldName)];
 
+        if (
+            !empty($options['label']) && $options['type'] != 'hidden' ||
+            !empty($options['placeholder']) && $options['type'] != 'hidden'
+        ) {
+            $isFieldError = $this->isFieldError($fieldName);
+            $options += [
+                'aria-label' => !empty($options['label']) ? $options['label'] : $options['placeholder'],
+                'aria-required' => $options['required'] == true ? 'true' : null,
+                'aria-invalid' => $isFieldError ? 'true' : null,
```

Comment:
```suggestion
                'aria-invalid' => $isFieldError ? true : null,
```

You should be able to use a boolean here. If not we can fix up the attribute generation code.

Reply:
It should be true or false:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-invalid_attribute